File: python_programming_basics/conditional_statements_advanced/e_02_summer_outfit.py
degrees = int(input())
day_time = input()
outfit = ""
shoes = ""

if day_time == "Evening":
    outfit = "Shirt"
    shoes = "Moccasins"

elif day_time == "Morning":
    if 10 <= degrees <= 18:
        outfit = "Sweatshirt"
        shoes = "Sneakers"

    elif 18 < degrees <= 24:
        outfit = "Shirt"
        shoes = "Moccasins"

    else:
        outfit = "T-Shirt"
        shoes = "Sandals"

elif day_time == "Afternoon":
    if 10 <= degrees <= 18:
        outfit = "Shirt"
        shoes = "Moccasins"

    elif 18 < degrees <= 24:
        outfit = "T-Shirt"
        shoes = "Sandals"

    else:
        outfit = "Swim Suit"
        shoes = "Barefoot"

print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")

File: JavaScript Basics/03. Conditional Statements Advanced/e-02-Summer-Outfit.js
function recommendOutfit(degrees, timeOfDay) {
    let outfit = "";
    let shoes = "";

    if (degrees >= 10 && degrees <= 18) {
        if (timeOfDay === "Morning") {
            outfit = "Sweatshirt";
            shoes = "Sneakers";
        } else if (timeOfDay === "Afternoon" || timeOfDay === "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
    } else if (degrees > 18 && degrees <= 24) {
        if (timeOfDay === "Morning" || timeOfDay === "Afternoon") {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (timeOfDay === "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
    } else if (degrees >= 25) {
        if (timeOfDay === "Morning") {
            outfit = "T-Shirt";
            shoes = "Sandals";
        } else if (timeOfDay === "Afternoon") {
            outfit = "Swim Suit";
            shoes = "Barefoot";
        } else if (timeOfDay === "Evening") {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
    }

    console.log(`It's ${degrees} degrees, get your ${outfit} and ${shoes}.`);
}


recommendOutfit(16, "Morning");
recommendOutfit(22, "Afternoon");
recommendOutfit(28, "Evening");
